### PR TITLE
update Page and Accordion

### PR DESCRIPTION
**Pull request by rayhan-ab**

### CODE REVIEW

File: src/components/Accordion/index.tsx [hunk 1]
```diff
@@ -17,9 +17,10 @@ export interface AccordionProps {
   className?: string;
   /** Unique id passed to data-qa-id property of the outermost wrapper of the component. Used for automation testing  */
   dataQa?: string | null;
+  noPadding?: boolean;
 }
 
-export const Accordion = ({ title, className, children, dataQa }: AccordionProps) => {
+export const Accordion = ({ title, className, children, dataQa, noPadding = false }: AccordionProps) => {
```
Comment (aziz-zen-ab) [inline src/components/Accordion/index.tsx:23]: Is it still necessary to put `false` value even though the `noPadding` props is already optional props?

File: src/components/Page/Page.tsx [hunk 2]
```diff
@@ -27,6 +29,8 @@ export const Page = ({
   dataQa,
   headerRightElement,
   noSidebar = false,
+  noMarginBottom = false,
```
Comment (aziz-zen-ab) [inline src/components/Page/Page.tsx:32]: same question as above

File: src/components/Accordion/index.tsx (hunk 1 above)
Comment (rayhan-ab) [reply src/components/Accordion/index.tsx:23]: just wanna make it clearer in the docs. the props table has a "Default" column that lists the default value of each props

Comment (aziz-zen-ab) [reply src/components/Accordion/index.tsx:23]: I see

**Review by aziz-zen-ab — APPROVED** [review]

**Pull request merged**